Pal/src/host/Linux-SGX/enclave_framework.c

```diff
@@ -17,6 +17,8 @@ void * enclave_base, * enclave_top;
 
 struct pal_enclave_config pal_enclave_config;
 
+static int register_trusted_file (const char * uri, const char * checksum_str);
```

Comment:
@rainfld can you add a short comment for this function, explaining what it does, and what the arguments are.  It is somewhat obvious, but we are trying to (incrementally) beef up the code documentation.